packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -66,6 +66,7 @@ export default deepFreeze(
         gaID: null,
         trackErrors: true,
       },
+      topTagPrefixes: ['http.'],
```

Comment:
the priority sorting feels contrary to the intention of this PR. Is it really necessary?

Reply:
Sorting by itself makes it much easier to find relevant information in a trace compare to experience with tags in the random order. At least the tag one is trying to follow is at the same position in every span. 

I was trying to find a way to access tags that that are more relevant than others faster. Issue #218 mentions that and in my installation I have a similar problem
<img width="975" alt="Screenshot 2019-12-02 at 10 01 14" src="https://user-images.githubusercontent.com/751681/69945863-c0563780-14ea-11ea-9b98-3b4339e7645e.png">
Without priority sorting I would have to name tags the way they are alphabetically in order of importance to get most relevant ones in that view.

Though I bet there are more elegant solutions for that than priority sorting. I think it's quite a standard approach to have many tags some of which are used rarely for things like troubleshooting corner cases and others are crucial for most of the usage scenarios.

Reply:
I like the priority sorting as a config option, but I wouldn't add a default value.